src/modules/chat/style.css

```diff
@@ -33,6 +33,14 @@
   transform: scaleY(-1);
 }
 
+.bttv-emote-modifier-rotate-left img {
```

Comment:
<img width="478" alt="Screenshot 2023-05-13 at 16 32 18" src="https://github.com/night/betterttv/assets/43322006/ec199c09-4b31-4dd8-9baa-6cd9c98c22c8">

Overlapping isn't ideal, can we add a max-width when applying this style?

e.g. `max-width: 28px;`

Reply:
added